### DISCUSSION TITLE
Google security update September 13, 2021

### DISCUSSION THREAD

**Discussion by malinsky82** [nithinmurali/pygsheets#502] [Q&A] [answered]
I have received some email from Google regarding a Google Drive security update on September 13, 2021. I was wondering if it could impact the pygsheets package!?

**Comment by nithinmurali** [comment, marked as answer]
Essentially Google is changing the format of the shared file URLs by adding a resource key in the url.
since pygsheets doesn't directly deal with shared URLs, I don't think there would be any effect. Also, I don't see any proposed changes in [drive api](https://developers.google.com/drive/api/v3/manage-sharing).
